eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Epoch.hs

```diff
@@ -225,7 +218,7 @@ epochTransition = do
     oblgNew = totalObligation adjustedCertState (utxoSt'' ^. utxosGovStateL)
     utxoSt''' = utxoSt'' {utxosDeposited = oblgNew}
   pure $
-    epochState'
+    EpochState acnt' ls' ss' nm
       & esAccountStateL .~ acnt'
```

Comment:
Is this line still necessary, since we're building `EpochState` with `acnt'` a line above?

Reply:
Nice catch. It is most certainly redundant